content/en/docs/zero-code/java/spring-boot-starter/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Spring Boot starter
+description: Spring Boot instrumentation for OpenTelemetry Java
+aliases: [/docs/languages/java/automatic/spring-boot/]
```

Comment:
You'll need to add an alias to the old path (without the `-starter` suffix).

Reply:
Added